ReactApp/src/AppRoutes.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useContext } from "react";
+
+import { Route, BrowserRouter, Routes } from "react-router-dom";
+
+/// demos
+import MainDashboard from 'React-Automation-Studio/components/UI/MainDashboard';
+import Vault from 'React-Automation-Studio/components/AlarmHandler/Vault';
+//system
+import Login from "React-Automation-Studio/components/SystemComponents/Login";
+import Probe from "React-Automation-Studio/components/SettingsPages/Probe";
+import Help from "React-Automation-Studio/components/docs/Help";
+import AppMain from "./AppMain";
+import AutomationStudioContext from "React-Automation-Studio/components/SystemComponents/AutomationStudioContext";
+import Administrator from "React-Automation-Studio/components/Administrator/Administrator";
+import UserProfile from "React-Automation-Studio/components/SystemComponents/userProfiles/UserProfile";
+import ProtectedRoute from "React-Automation-Studio/components/SystemComponents/ProtectedRoute";
+
+
+const AppRoutes = (props) => {
+  const context = useContext(AutomationStudioContext);
+  /* eslint-disable-next-line no-unused-vars */
+  const userData = context.userData;
+  /* eslint-disable-next-line no-unused-vars */
+  const roles = context.userData.roles;
+  /* eslint-disable-next-line no-unused-vars */
+  const username = context.userData.username;
+  let enableDemos =
+    typeof import.meta.env.VITE_DISABLE_DEMOS !== "undefined"
+      ? !(import.meta.env.VITE_DISABLE_DEMOS.toUpperCase() === "TRUE")
+      : true;
+
+  return (
+    <BrowserRouter>
+      <Routes>
+      <Route
+          exact
+          path="/"
+          element={
+            <ProtectedRoute>
+              <AppMain/>
+            </ProtectedRoute>
+          }
+        />
+       <Route
+          exact
+          path="/MainDashboard"
+          element={
+            <ProtectedRoute>
+              <MainDashboard />
+            </ProtectedRoute>
+          }
+        />
+        <Route
+          exact
+          path="/Administrator"
+          element={
+            <ProtectedRoute roles={["admin"]}>
+              <Administrator />
+            </ProtectedRoute>
+          }
+        />
+        {/* <ProtectedRoute path="/UserProfile" component={UserProfile} /> */}
```

Comment:
Is there value in keeping this line here? If so, would it be possible to add a comment describing the intention (e.g. "Use this to ..."), please?